examples/simple-game-server/Dockerfile.windows

```
@@ -16,16 +16,25 @@
 # Use `--build-arg WINDOWS_VERSION=` to select the correct base image
 # See https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility
 # for details.
+
+# Build Stage
 ARG WINDOWS_VERSION=ltsc2019
 
+FROM golang:1.20.4-windowsservercore as builder
+SHELL ["powershell", "-command"]
```

```suggestion
FROM --platform=linux/amd64 golang:1.20.4 as base
```

Is what it should be, since we can't run windows containers on Linux.

---

I've updated the platform to the linux/amd64 architecture. According to the log, it's recommended to include the pkg/sdk, pkg/util/signals, and sdks/go modules. However, I've noticed that the files remain unchanged when I test this locally. Could you please review this [build](https://pantheon.corp.google.com/cloud-build/builds;region=global/6fff24b9-b742-4ba2-84aa-b737c14434ac;step=2?mods=logs_tg_prod&project=agones-mangalpalli)?